### ISSUE TITLE
Compatibility with Dask.DataFrame

### ISSUE THREAD

**Issue by endu** [pandas-profiling/pandas-profiling#279]
I want to use pandas-profiling with Dask.DataFrame.

Now, when I use Dask.DataFrame, pandas-profiling shows error message.

```
dd_data = dd.read_csv('data.csv',assume_missing=True)
pr_data = pp.ProfileReport(dd_data)
:
/usr/lib/python3.6/site-packages/pandas_profiling/utils/dataframe.py in rename_index(df)
     93         The DataFrame with {index,column} `index` replaced by `df_index`, unchanged if the DataFrame does not contain such a string.
     94     """
---> 95     df.rename(columns={"index": "df_index"}, inplace=True)
     96 
     97     if "index" in df.index.names:

TypeError: rename() got an unexpected keyword argument 'inplace'
```

Is it difficult to have compatibility with Dask.DataFrame ?

**Comment by sbrugman** [comment]
Making `pandas-profiling` compatible with the Dask.DataFrame would be an enormeous operation. My opinion is that the package shouldn't introduce dependencies by doing too much.

However, luckily @devin-petersohn is working on precisely that. We are eagerly waiting for it's stable release. If you haven't already, check out the project here:

https://github.com/modin-project/modin

**Comment by endu** [comment]
I've acknowledged. And, I did not know modin-project. That looks very nice!
Thanks for your response.

**Comment by brunompacheco** [comment]
I've developed some work on modifying pandas-profiling to make it work for dask dataframes and successfully generated some reports.

I've developed it in the company I work for (therefore, the code is currently on our private repo), but I already received the green light to publish the code, just need to mention them. Also, since this stayed in the low-priority side of my tasks for a long time, the fork is from version 1.4.2, but I'm looking forward to update it to the current version.

By the way, should I move this to #227?
